### DISCUSSION TITLE
Want push notifications when someone posts an issue on my repos

### DISCUSSION THREAD

**Discussion by sacredbanana** [community/community#113135] [Mobile]
### Select Topic Area

Product Feedback

### Body

It would be great if the mobile app would give you a push notification when someone posts an issue on one of your repos. This way I can respond to them as fast as possible.

**Comment by esahmed21** [comment]
Thank you for getting in touch, @sacredbanana! I appreciate your input and will certainly bring your suggestion to the team for further consideration and discussion.
